modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java

```diff
@@ -182,8 +177,7 @@ private void startUnRegistration() {
 				                    getResources().getString(R.string.dialog_message_please_wait),
 				                    true);
 
-		regId =
-				Preference.getString(context,
+		regId = Preference.getString(context,
```

Comment:
context.getResources().getString(R.string.shared_pref_regId)) can be brought up to this line.